nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/jquery/propertytable/jquery.propertytable.js

```diff
@@ -2001,6 +2020,45 @@
         propertyData.setItems([]);
     };
 
+    /**
+     * Gets all the referenced parameters from the {parameterContext} based on the value of {parameterReference} with matching {sensitive} property
+     *
+     * @param {string} parameterReference
+     * @param {ParameterContextEntity} parameterContext
+     * @param {boolean} sensitive
+     * @returns {ParameterEntity[]}
+     */
+    var getExistingParametersReferenced = function (parameterReference, parameterContext, sensitive) {
```

Comment:
Is there a reason for passing `parameterContext` and `sensitive` as arguments? It looks like `sensitive` is always `false` and `parameterContext` always references `currentParameterContext`. Unless there is some other reason to keep them, it seems like the implementation could be simplified.

Reply:
@exceptionfactory I've simplified the function by removing the `sensitive` and `parameterContext` parameters.